docs/spelling_wordlist.txt

```diff
@@ -16,6 +16,7 @@ backported
 biome
 checkbox
 contrib
+csp
```

Comment:
Do we need this?

Reply:
Yes, I think my note in the changelog triggered it as a word despite being `django-csp`